### ISSUE TITLE
Cleanup cleans up all pins

### ISSUE THREAD

**Issue by paf0186** [milaq/rpi-rf#61]
https://github.com/milaq/rpi-rf/blob/71fbe0507588cadd0e58978d2ed72a3cb5994f16/rpi_rf/rpi_rf.py#L69

This cleans up all the pins, making it not safe to call if you've got anything else using a GPIO pin.

cleanup() supports specifying a specific GPIO pin (or set of pins as a tuple), so this should be an easy fix.

**Comment by RootnuII** [comment]
I don't think it does that much I'm probably wrong what do you mean by not safe I don't understand
